source/includes/_lips_contact_interactions.md

```diff
@@ -159,7 +173,15 @@ curl -L -g -X GET 'https://data-connect-lips.ganettdigital.com/contact_interacti
     ],
     "page": 1,
     "total_pages": 1,
-    "per_page": 25
+    "per_page": 25,
+    "total_events": 3,
+    "total_call_events": 3,
+    "total_form_events": 0,
+    "total_chat_events": 0,
+    "total_unread_events": 2,
+    "total_unread_call_events": 2,
+    "total_unread_form_events": 0,
+    "total_unread_chat_events": 0
```

Comment:
I'm not a fan of this structure.  What would you think about changing the structure to something like this
```
"totals": {
  "total": 3,
  "call" 3,
  "form": 0,
  "chat": 0
},
"unread": {
  "total": 2,
  "call" 2,
  "form": 0,
  "chat": 0
}
```

Reply:
Noted, I will take care